tiered-storage/jcloud/src/test/java/org/apache/bookkeeper/mledger/offload/jcloud/BlobStoreBackedInputStreamTest.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.bookkeeper.mledger.offload.jcloud;
 
+import static junit.framework.TestCase.assertEquals;
```

Comment:
Use testng instead of junit?

Reply:
sure, my IDE chose the wrong `assertEquals`